src/common/imageio_rawspeed.cc

```diff
@@ -120,7 +120,7 @@ dt_imageio_retval_t dt_imageio_open_rawspeed(dt_image_t *img, const char *filena
 
 #ifdef __WIN32__
   const size_t len = strlen(filename) + 1;
-  wchar_t filen[len];
+  wchar_t filen[MAX_PATH];
   mbstowcs(filen, filename, len);
```

Comment:
@LebedevRI : I agree that this piece needs to be reviewed. The change above was merely removing a compiler warning. There are lot of other code pieces which are enclosed in some shape or form in conditional compiles like `_WIN32` or `__WIN32__ `or `G_OS_WIN32`. First I need to unify them using at least the same conditional, and then carefully review all as these code parts, as most likely they were not (fully) tested.
The code `mbstowcs(filen, filename, len);` has been there for a while. I believe even `sizeof(filen) / sizeof(wchar_t)` would not be OK, as multibyte character set may consist of both one-byte and two-byte characters (see [here](https://msdn.microsoft.com/en-us/library/4bb3e64h.aspx)), so if filename is really a multi-byte string (which I'm not fully convinced), then the best is to get the size is using first a call to determine the size: `requiredSize = mbstowcs(NULL, filename, 0);`

This particular code block is in the function called `dt_imageio_open_rawspeed` I have also noticed that sibling functions like `dt_imageio_open_jpeg` or `dt_imageio_open_png` or `dt_imageio_open_exr` have no similar conditional code blocks handling MB to WC conversions, so we might need an approach which would work also for all these functions, if filename is indeed a MB string.